screens/product-listing-page/product-listing-page.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Products | Need Games</title>
+		<link rel="stylesheet" href="../../styles.css" />
+	</head>
+	<body class="product-homepage">
+		<nav class="homepage-navbar">
+			<div class="navbar-brand">
+				<a href="index.html"> <h1>Need Games</h1> </a>
+			</div>
+			<input type="search" class="text-field" />
+			<div class="navbar-cta">
+				<div class="badge-holder flex-column">
+					<svg
+						xmlns="http://www.w3.org/2000/svg"
+						xmlns:xlink="http://www.w3.org/1999/xlink"
+						aria-hidden="true"
+						role="img"
+						class="iconify iconify--ph"
+						width="32"
+						height="32"
+						preserveAspectRatio="xMidYMid meet"
+						viewBox="0 0 256 256"
+					>
+						<path
+							d="M221.7 72l-26.4 92.4a15.9 15.9 0 0 1-15.4 11.6H84.1a15.9 15.9 0 0 1-15.4-11.6L42.3 72z"
+							opacity=".2"
+							fill="currentColor"
+						></path>
+						<path
+							d="M96 216a16 16 0 1 1-16-16a16 16 0 0 1 16 16zm88-16a16 16 0 1 0 16 16a16 16 0 0 0-16-16zm45.4-125.8L203 166.6a24.1 24.1 0 0 1-23.1 17.4H84.1A24.1 24.1 0 0 1 61 166.6L34.6 74.3v-.2L24.8 40H8a8 8 0 0 1 0-16h16.8a16.1 16.1 0 0 1 15.4 11.6L48.3 64h173.4a8 8 0 0 1 7.7 10.2zM211.1 80H52.9l23.5 82.2a8 8 0 0 0 7.7 5.8h95.8a8 8 0 0 0 7.7-5.8z"
+							fill="currentColor"
+						></path>
+					</svg>
+					<div class="badge-icon">1</div>
+				</div>
+
+				<div class="badge-holder flex-column">
+					<svg
+						xmlns="http://www.w3.org/2000/svg"
+						xmlns:xlink="http://www.w3.org/1999/xlink"
+						aria-hidden="true"
+						role="img"
+						class="iconify iconify--ph"
+						width="32"
+						height="32"
+						preserveAspectRatio="xMidYMid meet"
+						viewBox="0 0 256 256"
+					>
+						<path
+							d="M128 216S28 160 28 92a52 52 0 0 1 100-20a52 52 0 0 1 100 20c0 68-100 124-100 124z"
+							opacity=".2"
+							fill="currentColor"
+						></path>
+						<path
+							d="M128 224a7.8 7.8 0 0 1-3.9-1C119.8 220.6 20 163.9 20 92a60 60 0 0 1 108-36a60 60 0 0 1 108 36c0 30.6-17.7 62-52.6 93.4a314.3 314.3 0 0 1-51.5 37.6a7.8 7.8 0 0 1-3.9 1zm-3.9-15zM80 48a44 44 0 0 0-44 44c0 55.2 74 103.7 92 114.7c18-11 92-59.5 92-114.7a44 44 0 0 0-84.6-17a8 8 0 0 1-14.8 0A43.8 43.8 0 0 0 80 48z"
+							fill="currentColor"
+						></path>
+					</svg>
+					<div class="badge-icon">9</div>
+				</div>
+				<img
+					class="avatar avatar-sm"
+					src="https://media-exp1.licdn.com/dms/image/C4E03AQEm0ZBjaIr1hg/profile-displayphoto-shrink_200_200/0/1610261504692?e=1649894400&v=beta&t=kKO77EwqflEDsrZf5eG7xC7ZUB4hD_BkZuRJyFiFHbI"
+				/>
+			</div>
+		</nav>
+		<aside class="product-aside"> </aside>
+		<main class="product-main">
+			<div class="card">
+				<div class="card-overlay-holder">
```

Comment:
try not using pngs, jpgs and webps are lot lighter files

Reply:
png is used for transparent background.